### ISSUE TITLE
Basic Link + Ad: Ad Start and End Dates

### ISSUE THREAD

**Issue by jakebrenner** [NiteshOswal/ezrtv3#31]
Below the Ad End Date there should be small text that says "Minimum 7 day duration"

**Comment by NiteshOswal** [comment]
Okie,  I will enforce it too?

**Comment by jakebrenner** [comment]
Yes please, this is a big part of people triggering the Facebook #100 error is when they don't have a long enough duration

**Comment by NiteshOswal** [comment]
Sorry didn't wanted to add a?
